docs/editors/vscode.md

```diff
@@ -132,6 +132,65 @@ As you type, the symbol outline is also visible at the top of the file.
 
 ```
 
+## Running and debugging your code
+
+Metals supports running or debugging tests and main methods via
+[Debug Adapter Protocol](https://microsoft.github.io/debug-adapter-protocol/),
+used to communicate with the editor, and
+[Bloop](https://scalacenter.github.io/bloop/), which is responsible for starting
+the actual process. This means that applications can be run the same as for any
+other language in the natively supported `Run` view.
+
+Users can begin the debugging session in two ways:
+
+- via code lenses
+
+For each main or test class Metals shows two code lenses `run | debug`, which
+shows up above the definition as a kind of virtual text. Clicking `run` will
+start running the test or main class without stopping at any breakpoints, while
+clicking `debug` will pause once any of them is hit. It's not possible to add
+any arguments or java properties when running using this method.
+
+- via `launch.json` configuration
+
+Visual Studio Code uses `.vscode/launch.json` to store user defined
+configurations, which can be run using:
+
+- `Run -> Start Debugging` or `workbench.action.debug.start` shortcut.
+- `Run -> Run Without Debugging` or `workbench.action.debug.run` shortcut.
+
+If a user doesn't have anything yet saved, a configuration wizard will pop up to
+guide them. In the end users should end up with something like this:
+
+```json
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "scala",
+      "request": "launch",
+      // configuration name visible for the user
```

Comment:
It should, VS Code does allow for it.

Reply:
> Can we attach this image to show how lenses should look like?

Added it!

Reply:
I think it will also be displayed here: https://scalameta.org/metals/docs/editors/vscode.html

We can see how it looks like after merge and quickly update if something will not be right.

Reply:
For rendering it correctly, I think you can use the good old trick of changing the code block language to `js` (same syntax, and it allows comments)